src/main/scala/com/monsanto/arch/cloudformation/model/resource/ApplicationAutoScaling.scala

```diff
@@ -26,10 +26,27 @@ object ApplicationAutoScaling {
     implicit val format: JsonFormat[T] = jsonFormat5(apply)
   }
 
-  sealed trait ScalingType extends Product with Serializable
+  sealed trait ServiceNamespace extends Product with Serializable
+
+  object ServiceNamespace extends DefaultJsonProtocol {
+    private type T = ServiceNamespace
+    case object appstream extends T
+    case object `custom-resource` extends T
+    case object dynamodb extends T
+    case object ec2 extends T
+    case object ecs extends T
+    case object elasticmapreduce extends T
+    case object kinesis extends T
```

Comment:
I don't see kinesis in the list of valid values at https://docs.aws.amazon.com/autoscaling/application/APIReference/API_RegisterScalableTarget.html

Reply:
It was in the list when I wrote this. It must have been removed since then.